packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -333,9 +339,8 @@ class PointCloudStore {
     }
 
     const polygon = originPolygon;
-    const originPoints = this.scene.getObjectByName(this.pointCloudObjectName) as THREE.Points;
 
-    const cloudDataArrayLike = originPoints?.geometry?.attributes?.position?.array;
+    const cloudDataArrayLike = this.originPoints;
```

Comment:
What issue did you fix?

Reply:
origin pointcloud data was removed in focus mode, so you can't add or remove points in focus mode. 
Now i add origin points into store cache when loading pcd file to fix this problem